### PR TITLE
Fixes #1321: Removes pic.twitter.com/imageId links from tweet text

### LINKED ISSUE #1321: Removing Unusual link in the tweet text
- Issue type: Bug report

### Short description
 For the tweets with images, tweet text generally includes a link  `https://pic.twitter.com/imageID`. This link serves no purpose as it redirects to the tweet itself. This link should not be included with the tweet text as it serves no purpose. This link is also hidden on the twitter itself.

screen shot 2017-07-14 at 8 55 49 pm

### PR DESCRIPTION
**Pull request by Achint08**
### Short description

Fixes #1321.
- Removes `pic.twitter.com/ImageId` links from tweet text

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by Achint08** [comment]
Open for review @mariobehling @singhpratyush @vibhcool @hemantjadon @SKrPl @djmgit @kavithaenair

File: src/org/loklak/harvester/TwitterScraper.java [hunk 1]
```diff
@@ -811,7 +811,7 @@ public static String unshorten(String text) {
                 Matcher m = timeline_embed_pattern.matcher(text);
                 if (m.find()) {
                     String shorturl = RedirectUnshortener.unShorten(m.group(2));
-                    text = m.replaceFirst(" https://pic.twitter.com/" + shorturl + " ");
+                    text = m.replaceFirst("");
                     continue;
```
Comment (vibhcool) [inline src/org/loklak/harvester/TwitterScraper.java]: I haven't tried but this entire try catch block shall be removed

Comment (Achint08) [inline src/org/loklak/harvester/TwitterScraper.java]: Done! :)

File: src/org/loklak/harvester/TwitterScraper.java [hunk 2]
```diff
@@ -810,8 +810,7 @@ public static String unshorten(String text) {
             try {
                 Matcher m = timeline_embed_pattern.matcher(text);
```
Comment (vibhcool) [inline src/org/loklak/harvester/TwitterScraper.java]: also you need to remove this method's implementation at here ->
https://github.com/Achint08/loklak_server/blob/82357c2671496fee9bd7a1ff8ceea3ed3c26f898/src/org/loklak/harvester/TwitterScraper.java#L723

please also check in other lines, if i missed

Comment (Achint08) [inline src/org/loklak/harvester/TwitterScraper.java]: Done! 
I checked it. This was the only place. :)

**Pull request merged**